_posts/2024-10-14-enhancing-quarkus-developer-experience.adoc

```diff
@@ -0,0 +1,10 @@
+---
+layout: post
+title: 'Enhancing the Quarkus developer experience: New updates for IntelliJ and VS Code tools'
+synopsis: We wrote an article on the Red Hat Developer blog about new features in the Quarkus tooling for IntelliJ and VS Code
+date: 2024-10-14
+tags: ide
+author: rgrunber
+---
+
+With the recent releases of "Quarkus Tools for IntelliJ" https://github.com/redhat-developer/intellij-quarkus/releases/tag/2.0.2[2.0.2], and "Quarkus Tools for Visual Studio Code" https://github.com/redhat-developer/vscode-quarkus/releases/tag/1.18.1[1.18.1] we wrote an article at https://developers.redhat.com/articles/2024/09/18/enhancing-quarkus-developer-experience-new-updates-intellij-and-vs-code-tools describing some of the new features available in both extensions. Be sure to check it out!
```

Comment:
I will defer to @maxandersen and @insectengine but I'm not sure we want to host blog posts that are just pointers to other websites.

Reply:
I agree. Can we have at least a summary of these 2 posts. Linking them no problem, but here we need more content.